ERKER2Phenopackets/data/out/phenopackets/example-phenopackets-from-synthetic-data/0.json

```diff
@@ -36,16 +56,12 @@
                 "expressions": [
                   {
                     "syntax": "hgvs",
-                    "value": "NP_005903.2:p.Thr112Met"
-                  },
-                  {
-                    "syntax": "hgvs",
-                    "value": "NM_005912.3:c.307G>A"
+                    "value": "NP_005903.2:p.Ile251Leu"
```

Comment:
There is a small issue in the protein HGVS string.

The [HGVS Protein recommendation](https://varnomen.hgvs.org/recommendations/protein/variant/substitution/) prescribes putting predicted consequences into parentheses:

> predicted consequences, i.e. without experimental evidence (no RNA or protein sequence analysed), should be given in parentheses, e.g. p.(Arg727Ser)

Correct me if I'm wrong, but the protein effects are predicted in your data (the protein variants were not determined experimentally, e.g. with proteomic methods). Therefore, you should use

```
NP_005903.2:p.(Ile251Leu)
```
for the example above as well as (probably) for the other phenopackets.